src/test/java/com/demoqa/shop/steps/UpdateAccountStepdefs.java

```diff
@@ -0,0 +1,265 @@
+package com.demoqa.shop.steps;
+
+import com.demoqa.shop.pages.*;
+import com.demoqa.shop.util.*;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.openqa.selenium.*;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class UpdateAccountStepdefs {
+
+    Logger log = LoggerFactory.getLogger(UpdateAccountStepdefs.class);
+    public WebDriver driver = Browser.getBrowser();
+
+    @Given("Demoqa shop homepage is opened")
+    public void demoqaShopHomepageIsOpened() {
+        HomePage homePage = (HomePage) ScenarioContext.getInstance().getContext(Context.CURRENT_PAGE);
+        ATFAssert.assertEquals(homePage.getTitle(), driver.getTitle(), "New page is opened", "Page" + homePage.getUrl() + " is opened");
+
+        // assertEquals(homePage.getTitle(), driver.getTitle());
+        log.info("Home page with url {}  is loaded",
+                driver.getCurrentUrl());
+    }
+
+    @And("he clicks account link menu")
```

Comment:
use neutral "user" instead of "he" or "she"

Reply:
make the same changes for the rest of the steps

Reply:
"he" was relapsed with "user"